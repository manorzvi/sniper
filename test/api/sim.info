{'cmdline': ['../../run-sniper', '-n', '1', '-c', 'gainestown', '--', './api'],
 'git_revision': '',
 'host': 'butler',
 'pin_version': 'unknown',
 'rusage': (7.679346,
            0.039999,
            32904,
            0,
            0,
            0,
            6677,
            0,
            0,
            0,
            144,
            0,
            0,
            0,
            7618,
            14),
 'snipercmd': ['bash',
               '-c',
               '/home/manorz/sniper-7.4/lib/sniper -c /home/manorz/sniper-7.4/config/base.cfg --general/total_cores=1 --general/output_dir=/home/manorz/sniper-7.4/test/api --config=/home/manorz/sniper-7.4/config/nehalem.cfg --config=/home/manorz/sniper-7.4/config/gainestown.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --traceinput/stop_with_first_app=false -g --traceinput/enabled=true -g --traceinput/emulate_syscalls=true -g --traceinput/num_apps=1 -g --traceinput/trace_prefix=/tmp/tmp_tsWy5/run_benchmarks'],
 't_elapsed': 10.575257062911987,
 't_start': 1633236044.590566,
 'tracecmds': [['/home/manorz/sniper-7.4/record-trace',
                '-o',
                '/tmp/tmp_tsWy5/run_benchmarks',
                '--pa',
                '-e',
                '1',
                '-s',
                '0',
                '-r',
                '1',
                '--follow',
                '--routine-tracing',
                '--',
                './api']],
 'user': 'root',
 'vmem': 220901376L}
