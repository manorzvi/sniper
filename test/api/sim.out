                                     | Core 0    
  Instructions                       |    5039385
  Cycles                             |   12106192
  IPC                                |       0.42
  Time (ns)                          |    4551200
  Idle time (ns)                     |        -57
  Idle time (%)                      |      -0.0%
Branch predictor stats               |           
  num correct                        |    1009216
  num incorrect                      |       1060
  misprediction rate                 |      0.10%
  mpki                               |       0.21
TLB Summary                          |           
  I-TLB                              |           
    num accesses                     |       4813
    num misses                       |         46
    miss rate                        |      0.96%
    mpki                             |       0.01
  D-TLB                              |           
    num accesses                     |    6009470
    num misses                       |         21
    miss rate                        |      0.00%
    mpki                             |       0.00
  L2 TLB                             |           
    num accesses                     |         67
    num misses                       |         67
    miss rate                        |    100.00%
    mpki                             |       0.01
Cache Summary                        |           
  Cache L1-I                         |           
    num cache accesses               |       4813
    num cache misses                 |        470
    miss rate                        |      9.77%
    mpki                             |       0.09
  Cache L1-D                         |           
    num cache accesses               |    6009470
    num cache misses                 |        299
    miss rate                        |      0.00%
    mpki                             |       0.06
  Cache L2                           |           
    num cache accesses               |        794
    num cache misses                 |        764
    miss rate                        |     96.22%
    mpki                             |       0.15
  Cache L3                           |           
    num cache accesses               |        789
    num cache misses                 |        764
    miss rate                        |     96.83%
    mpki                             |       0.15
DRAM summary                         |           
  num dram accesses                  |        764
  average dram access latency (ns)   |      83.92
  average dram queueing delay        |      30.50
  average dram bandwidth utilization |      0.00%
Coherency Traffic                    |           
  num loads from dram                |        165
  num loads from dram cache          |          0
  num loads from remote cache        |          0
